core/services/feeds/service.go

```
@@ -402,9 +468,7 @@ func (s *service) CreateChainConfig(ctx context.Context, cfg ChainConfig) (int64
 		return 0, errors.Wrap(err, "CreateChainConfig: failed to fetch manager")
 	}
 
-	if err := s.SyncNodeInfo(ctx, mgr.ID); err != nil {
-		s.lggr.Infof("FMS: Unable to sync node info: %v", err)
-	}
+	s.syncNodeInfoWithRetry(mgr.ID)
```

I see, this will be a behaviour change, previously `SyncNodeInfo` is synchronous but now it is async

---

Yes, but `SyncNodeInfo` could fail silently -- the FeedService caller wouldn't know any better -- the only side-effect of the failure was an error message.

With the new version, the caller sees the save behavior, just faster. The clo network requests happens in the background, and we continue to log any errors.